pyincore/analyses/buildingdamage/buildingdamage.py

```diff
@@ -41,12 +41,24 @@ def run(self):
         else:
             retrofit_strategy = None
 
-        # Accomodating to multihazard
-        # Get hazard input
-        hazard_dataset_ids = self.get_parameter("hazard_id").split("+")
-
-        # Hazard type of the exposure
-        hazard_types = self.get_parameter("hazard_type").split("+")
+        # Accommodating to multi-hazard
+        hazards = []  # hazard objects
+        hazard_object = self.get_input_hazard("hazard")
+
+        # To use local hazard
+        if hazard_object is not None:
+            # Right now only supports single hazard for local hazard object
+            hazard_types = [hazard_object.hazard_type]
```

Comment:
This is the part we can rewrite to include multiple local hazard objects in the future